tools/simple-test/tester.py

```diff
@@ -0,0 +1,13 @@
+import yaml
+import json
+import sys
+from jsonschema import validate
+
+schema_path = sys.argv[1]
+sample_path = sys.argv[2]
+
+with open(sample_path, 'r') as sample_data:
+    with open(schema_path, 'r') as schema_data:
```

Comment:
- _r_ is the default mode. ([reference](https://docs.python.org/ja/3.6/library/functions.html#open))
- _with_ can have multiple clauses and save indentation.

```suggestion
with open(sample_path) as sample_data, open(schema_path) as schema_data:
```